pkgs/development/compilers/elm/packages/ghc9_2/elm-format/avh4-lib.nix

```diff
@@ -22,7 +21,10 @@ mkDerivation {
     relude tasty tasty-hspec tasty-hunit text
   ];
   testToolDepends = [ tasty-discover ];
-  doHaddock = false;
   description = "Common code for haskell projects";
   license = lib.licenses.bsd3;
+
+  # Added manually
+  configureFlags = [ "--ghc-option=-Wno-error=unused-packages" ];
+  doHaddock = false;
```

Comment:
I think we can do these in overrides in ghc9_2/default.nix so we don't lose it if the file itself gets regenerated.

Reply:
Sounds much better.